docs/standard/serialization/system-text-json-converters-how-to.md

```diff
@@ -279,22 +280,25 @@ The converter can deserialize JSON that was created by using the same converter
 
 ## Other custom converter samples
 
-The [unit tests folder](https://github.com/dotnet/corefx/blob/master/src/System.Text.Json/tests/Serialization/) in the `System.Text.Json.Serialization` source code includes other custom converter samples, such as:
+The [Migrate from Newtonsoft.Json to System.Text.Json](system-text-json-migrate-from-newtonsoft-how-to.md) article contains samples of custom converters.
```

Comment:
This comment/link is outdated now.

Reply:
It does have examples of other converters, just not the three that are in this doc.

Reply:
Like what, and are they useful/different enough to provide value? Should the link point to the specific converter snippet?

Also, maybe re-word as:
```suggestion
The [Migrate from Newtonsoft.Json to System.Text.Json](system-text-json-migrate-from-newtonsoft-how-to.md) article also contains some samples of custom converters.
```